ex2/api/models/User.ts

```diff
@@ -0,0 +1,25 @@
+import { client } from "../db";
+
+export class UserModel {
+    id: string | null;
+    email: string;
+    password: string;
+
+    constructor(data: UserCreationData) {
+        this.id = null;
+        this.email = data.email;
+        this.password = data.password;
+    }
+
+    public async save(): Promise<UserModel> {
+        const users = await client.query<UserModel>('INSERT INTO Users(email, password) VALUES($1, $2) RETURNING *', [this.email, this.password])
+        return users.rows[0];
```

Comment:
I think just returning a result from the database is not safe, with changes to the structure on the database side, your application will receive incorrect data and strange errors.
I am sure that the approach using model mappers (as we do in front-end solutions when processing data from the API) would be perfect in this case too.
It is not necessary to do this now, just for your information for the future.